src/main/java/com/infomaniak/lib/core/utils/CoilUtils.kt

```diff
@@ -83,6 +82,10 @@ object CoilUtils {
 
                         addInterceptor(TokenInterceptor(it))
                         authenticator(TokenAuthenticator(it))
+
+                        // Add common interceptors after every other interceptor so everything is already setup correctly by other
+                        // previous interceptors. Especially needed by the custom interceptors like AccessTokenUsageInterceptor
+                        HttpClientConfig.addCommonInterceptors(this)
```

Comment:
This code is inside the `tokenInterceptorListener?.let { … }`, is it on purpose to execute it only if we have the `tokenInterceptorListener` ?

Reply:
Yeah no, this doesn't look right, it should indeed be called everytime and not only when we have a tokenInterceptor